tensorflow/tools/ci_build/horovod/gpu/nightly.sh

```diff
@@ -45,29 +44,30 @@ wget https://download.open-mpi.org/release/open-mpi/v4.0/openmpi-4.0.4.tar.gz
 tar xvf openmpi-4.0.4.tar.gz
 
 # Install gcc.
-sudo apt install --assume-yes build-essential
+sudo apt install --assume-yes build-essential git
 
 gcc --version
 
 cd openmpi-4.0.4
-./configure
+./configure --enable-orterun-prefix-by-default
 
 # Install open-mpi.
-sudo make all install
+make -j $(nproc) all
+make install
 export LD_LIBRARY_PATH=/usr/local/lib/openmpi
 sudo ldconfig
 
-sudo update-alternatives --install /usr/bin/gcc gcc /dt7/usr/bin/gcc 60 --slave /usr/bin/g++ g++ /dt7/usr/bin/g++
```

Comment:
your environment is not the same as ours. we need this to make sure that the right version of g++ is installed.